README.md

```diff
@@ -1 +1,116 @@
+following options defined:
+
+| Option             | Type     | Description |
+| ------------------ | -------- | ----------- |
+| `selectors` | `{Object}` | You must define one or more [selectors](http://docs.fluidproject.org/infusion/development/tutorial-gettingStartedWithInfusion/ViewComponents.html#selectors) that can be used in a binding. |
+| `bindings` | `{Object}` | Defines the relationship between selectors and model variables.  The full notation for this option is outlined below. |
+
+## Long notation
+
+There are two ways of specifying bindings.  The "long form" has named keys (as in the first example above) and
+supports the following options:
+
+* selector: A valid selector for your component.  Must be able to be resolved using `that.locate(selector)`
+* path: A valid path for the model variable whose value will be watched.  Must be able to be resolved using `fluid.get(that.model, path)`.
+
+The "long form" looks like:
+
+    bindings: {
+        "<key>": {
+            selector: "<selector1>",
+            path:     "<path1>"
+        }
+    }
+
+
+## Short notation
+
+The "short form" uses the selector as the key, and the path as a string value (as in the second example above).
+
+    bindings: {
+        "<selector2>": "<path2>"
+    }
+
+
+## Combining the two notations
+
+You can use both forms together, as in:
+
+    bindings: {
+        "<key>": {
+            selector: "<selector1>",
+            path:     "<path1>"
+        },
+        "<selector2>": "<path2>"
+    }
+
+
+
+# Static Functions
+
+## `gpii.binder.applyBinding(component)`
+* `component` `{Object}` - A fluid `viewComponent` with both `selectors` and `bindings` options defined (see above).
+* Returns: Nothing.
+
+You must explicitly invoke this function to create bindings.  Generally you wil do this from a
+[listener definition](http://docs.fluidproject.org/infusion/development/InfusionEventSystem.html#registering-a-listener-to-an-event).
+For example, if all required markup already exists on startup, you can simply bind to the "onCreate" event, as in:
+
+    listeners: {
+        "onCreate.applyBindings": {
+            "funcName": "gpii.binder.applyBinding",
+            "args":     "{that}"
+        }
+    }
+
+For an example of using the binder with static markup, see the tests in this package.
+
+### Bindings and dynamic markup
+
+If your component generates or regenerates markup, you will need to call `gpii.binder.applyBinding(component)`
+whenever it changes.  Best practice is to call `fluid.initDomBinder` when you have finished manipulating the DOM and
+then listen for the `viewComponent`'s `onDomBind` event, as in:
+
+    listeners: {
+        "onDomBind.applyBindings": {
+            "funcName": "gpii.binder.applyBinding",
+            "args":     "{that}"
+        }
+    }
+
+See the `templateAware` grade in [the `gpii-handlebars` package](https://github.com/GPII/gpii-handlebars) for a working
```

Comment:
Can we factor this templateAware grade into a part which goes into this package, including the above listener definition?